### PR TITLE
feature: Sphere management for debug

### PR DESCRIPTION
**Pull request by NikoSaul**

### CODE REVIEW

File: utils/debug/Debug.js [hunk 1]
```diff
@@ -436,7 +436,20 @@ function addGeometryLayerDebugFeatures(layer, view, gui, state) {
...
+                    const material = new THREE.MeshBasicMaterial({ wireframe: true });
+                    helper = new THREE.Mesh(geometry, material);
+                    node.add(helper);
+                    helper.frustumCulled = false;
+                    helper.layer = obb_layer_id;
+                    // add the ability to hide all the debug obj for one layer at once
+                    const l = context.view.getLayers(l => l.id === obb_layer_id)[0];
+                    const l3js = l.threejsLayer;
+                    helper.layers.set(l3js);
+                    helper.updateMatrix();
```
Comment (peppsac) [inline utils/debug/Debug.js]: This line is not needed in most case. See three.js updateMatrixWorld function:
```js
function updateMatrixWorld( force ) {
    this.matrixAutoUpdate && this.updateMatrix();
    ...
}
```

File: utils/debug/Debug.js [hunk 2]
```diff
@@ -436,7 +436,20 @@ function addGeometryLayerDebugFeatures(layer, view, gui, state) {
                     helper.updateMatrix();
                     helper.updateMatrixWorld();
                 }
-                // TODO deal with the bounding sphere case
+                if (node.boundingVolume.sphere) {
+                    const geometry = new THREE.SphereGeometry(node.boundingVolume.sphere.radius, 32, 32);
+                    const material = new THREE.MeshBasicMaterial({ wireframe: true });
+                    helper = new THREE.Mesh(geometry, material);
+                    node.add(helper);
+                    helper.frustumCulled = false;
```
Comment (peppsac) [inline utils/debug/Debug.js]: Is this line needed?

**Review by peppsac — COMMENTED** [review]
Can you post a screenshot to show this feature in action?

**Comment by NikoSaul** [comment]
Yes sure, the screen:
capture du 2017-07-10 14-33-58

We cannot see much, 
i added 1 to the radius of bounding volume of sphere to better see the bounding.
capture du 2017-07-10 14-30-09

**Review by peppsac — APPROVED** [review]

**Pull request merged**